tests/test/scheduler/test_mpi_exec_graph.cpp

```diff
@@ -36,8 +36,6 @@ TEST_CASE_METHOD(MpiTestFixture,
           rankA1, rankA2, BYTES(buffer), MPI_INT, messageData.size(), &status);
     }
 
-    REQUIRE(msg.intexecgraphdetails_size() == 1);
```

Comment:
I would have had to increase this number every time we tracked a different thing. It did not seem a useful thing to test/easy to forget to change.

Reply:
See point about constant above, I think this is worth testing.